log/rotation/file.go

```diff
@@ -83,11 +83,14 @@ func (f *File) write(p []byte) (int, error) {
 	}
 	if f.f == nil {
 		var err error
-		f.f, err = os.OpenFile(f.base, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0666)
+		f.f, err = os.OpenFile(f.base, os.O_WRONLY|os.O_CREATE|os.O_APPEND, 0644) // #nosec
```

Comment:
I had been assuming there would be a umask of 0002 or 0022 or 0077. (Or, if the operator really wants them to be world-writable, I suppose they could use 0 as the umask. But why would they ever want that.) This is safer for sure. :+1: